### ISSUE TITLE
Support IP accounting and access control for user units

### ISSUE THREAD

**Issue by anitazha** [systemd/systemd#12327]
We'd like to be able to do things like: `systemd-run --user --unit=boom -r -p IPAccounting=1 -p IPAddressDeny=any ping6 www.facebook.com` which currently is not supported for user units. I'm wondering how feasible adding this support would be (and of course am open to doing PRs if this sounds like a reasonable request).

**Comment by poettering** [comment]
Yeah, that would be great to have. But I think this is actually something that has to be done on the kernel side first, i.e. they'd need to open up the IP BPF hooks for unprivileged delegated userspace. If they'd do that I think this would just magically work since we don't add any additional restrictions.

I think it even was the intention of the kernel guys to open thus up eventually but they tried to be conservative initially?

**Comment by brandsimon** [comment]
Probably it is useful to warn users, if `IPAddressDeny` is specified but not enforced. Before I found this feature request, I wanted to report a bug, because it did not work as expected.
